guides/code-signing/code-signing-macos.md

```diff
@@ -62,44 +74,98 @@ To enable code signing on macOS, ensure that `packagerConfig.osxSign` exists in
 ```javascript
 module.exports = {
   packagerConfig: {
-    osxSign: {} // object must exist even if empty
+    osxSign: {
```

Comment:
Is there any reason not to recommend the empty configuration object as the default anymore?